test/feature2mesh_unit_test.js

```diff
@@ -55,4 +55,19 @@ describe('Feature2Mesh', function () {
                 noHoleArea - holeArea,
                 meshWithHoleArea);
         }));
+
+    it('square polygon should have double the vertices for being extruded', () =>
+        parse().then((feature) => {
+            const noExtrude = Feature2Mesh.convert()(feature[0]);
+            const extrude = Feature2Mesh.convert({
+                extrude: 2,
+            })(feature[0]);
+
+            const indexExtrude = new Uint16Array([5, 4, 7, 7, 6, 5, 0, 4, 1, 1,
```

Comment:
You're hardcoding in the test the ordering of the vertices. IMHO this isn't a good idea, because the test might fail if this order change - even if the output is correct.